### DISCUSSION TITLE
No CNN model from drop down menu

### DISCUSSION THREAD

**Discussion by JohnnyPicnic** [jomjol/AI-on-the-edge-device#1445] [General] [answered]
### The Problem

Brand new setup. No CNN model available for digital or analog.
Tried manually entering them in the config and that did not work.
Hmm maybe my sd card is not working correctly? There are no logs stored either...

### Version

13.0.01

### Logfile

```shell
"Failed to read existing file"
```

**Comment by caco3** [comment]
That really looks like your SD-Card is bad or the file system on it.
Can you take the [Web Console](https://jomjol.github.io/AI-on-the-edge-device/index.html) and check the USB log?

**Comment by caco3** [comment]
Others also have said that it was fixed after they formated the SD-Card with a different tool!

**Comment by JohnnyPicnic** [comment]
I did try formatting the card with several different tools. New card is on order.

**Comment by caco3** [comment]
the USB log would help to see what the cause could be!

**Comment by JohnnyPicnic** [comment, marked as answer]
Not sure what changed but I ran the OTA update a few times and all seems well.
Thank you!
![Screen Shot 2022-12-01 at 8 30 35 PM](https://user-images.githubusercontent.com/5553884/205209014-877dcaa0-615e-463b-b587-57744365c2c0.png)

**Reply by caco3** [reply]
Your camera focus could be better!

And it helps if the reference marks are as far apart as possible
